test/integration/statetest/CMakeLists.txt

```
@@ -131,6 +131,15 @@ set_tests_properties(
     FIXTURES_SETUP EXPORT_JSON_TESTS
 )
 
+add_test(
+    NAME ${PREFIX}/export_to_json_created_files
+    COMMAND ${CMAKE_COMMAND} -E sha256sum ${EXPORT_DIR}/state_transition/block/known_block_hash.json
```

this is just checking the existence of file? could be somethin like
```suggestion
    COMMAND ${CMAKE_COMMAND} -E test -f ${EXPORT_DIR}/state_transition/block/known_block_hash.json
```
?